test/integration/node-specific/operation_examples.test.ts

```diff
@@ -3394,103 +3397,94 @@ describe('Operation (Promises)', function () {
    */
-  it('shouldCorrectlyChangeProfilingLevelWithPromises', {
-    metadata: { requires: { topology: 'single' } },
-
-    test: function () {
-      var configuration = this.configuration;
-      var client = configuration.newClient(configuration.writeConcernMax(), { maxPoolSize: 1 });
-
-      return client.connect().then(function (client) {
-        var db = client.db(configuration.db);
-        // LINE var MongoClient = require('mongodb').MongoClient,
-        // LINE   test = require('assert');
-        // LINE const client = new MongoClient('mongodb://localhost:27017/test');
-        // LINE client.connect().then(() => {
-        // LINE   var db = client.db('test);
-        // REPLACE configuration.writeConcernMax() WITH {w:1}
-        // REMOVE-LINE restartAndDone
-        // REMOVE-LINE done();
-        // BEGIN
-
-        // Grab a collection object
-        var collection = db.collection('test_with_promise');
-        var adminDb = client.db('admin');
-
-        // Force the creation of the collection by inserting a document
-        // Collections are not created until the first document is inserted
-        return collection
-          .insertOne({ a: 1 }, { writeConcern: { w: 1 } })
-          .then(function (doc) {
-            test.ok(doc);
-            // Set the profiling level to only profile slow queries
-            return adminDb.setProfilingLevel('slow_only');
-          })
-          .then(function (level) {
-            test.ok(level);
-            // Retrieve the profiling level and verify that it's set to slow_only
-            return adminDb.profilingLevel();
-          })
-          .then(function (level) {
-            test.equal('slow_only', level);
-
-            // Turn profiling off
-            return adminDb.setProfilingLevel('off');
-          })
-          .then(function (level) {
-            test.ok(level);
-            // Retrieve the profiling level and verify that it's set to off
-            return adminDb.profilingLevel();
-          })
-          .then(function (level) {
-            test.equal('off', level);
-
-            // Set the profiling level to log all queries
-            return adminDb.setProfilingLevel('all');
-          })
-          .then(function (level) {
-            test.ok(level);
-            // Retrieve the profiling level and verify that it's set to all
-            return adminDb.profilingLevel();
-          })
-          .then(function (level) {
-            test.equal('all', level);
-
-            // Attempt to set an illegal profiling level
-            return adminDb.setProfilingLevel('medium');
-          })
-          .catch(function (err) {
-            test.ok(err instanceof Error);
-            test.equal(
-              `Profiling level must be one of "${enumToString(ProfilingLevel)}"`,
-              err.message
-            );
-            return client.close();
-          });
+  it('setProfilingLevel changes profiling level', async function () {
```

Comment:
Looks like this one is still failing as it's should only be running on a single and is failing CI on other topologies.